projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/VdkErrorMessage.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export class VdkErrorMessage {
+  public exception_message: String;
+  public what_happened: String;
+  public why_it_happened: String;
+  public consequences: String;
+  public countermeasures: String;
+
+  constructor(message: String) {
+    this.exception_message = '';
+    this.what_happened = '';
+    this.why_it_happened = '';
+    this.consequences = '';
+    this.countermeasures = '';
+
+    this.__parse_message(message);
+  }
+
+  private __parse_message(message: String): void {
+    const keys: (keyof VdkErrorMessage)[] = [
+      'exception_message',
+      'what_happened',
+      'why_it_happened',
+      'consequences',
+      'countermeasures'
+    ];
+    const delimiters = [
+      'ERROR : ',
+      'WHAT HAPPENED :',
+      'WHY IT HAPPENED :',
+      'CONSEQUENCES :',
+      'COUNTERMEASURES :'
```

Comment:
nit: I'd suggest removing the ' ' (blank space) after each delimiter (for example 'ERROR:'). I think this is the standard spacing practice.

Reply:
this is how the error is saved by vdk, that's why I use it with space